### ISSUE TITLE
Address a second I2C port

### ISSUE THREAD

**Issue by charlesbones** [firmata/ConfigurableFirmata#171]
Hi, I was wondering if it is possible to address a second I2C port ?

**Comment by pgrawehr** [comment]
No, unfortunately the protocol doesn't allow specifying a port for i2c communication. The only thing you can do is to patch the firmware to use another port as default. But then you can't access the first one.

**Comment by charlesbones** [comment]
Thank you for your reply
